### ISSUE TITLE
Use correct RTL stylesheet

### ISSUE THREAD

**Issue by justintadlock** [BenSibley/Chosen#14]
WordPress auto-loads a locale-specific CSS file and falls back to `rtl.css` if it's in the theme root.  It does this by calling `get_locale_stylesheet_uri()`.

However, your theme loads a custom RTL stylesheet, which bypasses the core-supported method.  That means that non-English users could run into issues with custom stylesheets.  It also means that plugins that filter `locale_stylesheet_uri` will not overwrite your stylesheet.

Basically, don't enqueue the RTL stylesheet.  Let WP do this.  That's the main thing.

From that point, you have one of two options: 1) move `styles/rtl.css` up one level or 2) filter `locale_stylesheet_uri` to point it to your current file.

**Comment by BenSibley** [comment]
This issue was moved to BenSibley/something#188
